rockcraft.yaml

```diff
@@ -11,9 +11,9 @@ platforms:
 parts:
   gnbsim:
     plugin: go
-    source: https://github.com/omec-project/gnbsim.git
+    source: https://github.com/Gmerold/gnbsim.git
```

Comment:
Now that the CLA is signed, I'd recommand to simply have the change made in the upstream project itself.